addons/account/models/account_report.py

```diff
@@ -653,4 +653,4 @@ class AccountReportExternalValue(models.Model):
     def _check_fiscal_position(self):
         for record in self:
             if record.foreign_vat_fiscal_position_id and record.foreign_vat_fiscal_position_id.country_id != record.report_country_id:
-                raise ValidationError(_("The country set on the the foreign VAT fiscal position must match the one set on the report."))
+                raise ValidationError(_("The country set on the foreign VAT fiscal position must match the one set on the report."))
```

Comment:
Need to update the translation files in stable

Reply:
Like this https://github.com/odoo/odoo/pull/107208/files#diff-c0fc2d3cf0a10147454a549ff13fd5968316dc86a8886db5e5e7f14798d0a0b8 ?